tests/WebAssertTest.php

```diff
@@ -1087,6 +1087,81 @@ public function testElementAttributeExists()
         );
     }
 
+    public function testElementAttributeNotExists()
+    {
+        $page = $this->getMockBuilder('Behat\\Mink\\Element\\DocumentElement')
+            ->disableOriginalConstructor()
+            ->getMock()
+        ;
+
+        $element = $this->getMockBuilder('Behat\\Mink\\Element\\NodeElement')
+            ->disableOriginalConstructor()
+            ->getMock()
+        ;
+
+        $this->session
+            ->expects($this->exactly(1))
+            ->method('getPage')
+            ->will($this->returnValue($page))
+        ;
+
+        $page
+            ->expects($this->exactly(1))
+            ->method('find')
+            ->with('css', 'h2 > span')
+            ->will($this->returnValue($element))
+        ;
+
+        $element
+            ->expects($this->exactly(1))
+            ->method('hasAttribute')
+            ->with('blah')
+            ->will($this->returnValue(false))
+        ;
+
+        call_user_func_array(array($this->assert, 'elementAttributeNotExists'), array('css', 'h2 > span', 'blah'));
+    }
+
+    public function testElementAttributeNotExistsThrowsException()
+    {
+        $page = $this->getMockBuilder('Behat\\Mink\\Element\\DocumentElement')
+            ->disableOriginalConstructor()
+            ->getMock()
+        ;
+
+        $element = $this->getMockBuilder('Behat\\Mink\\Element\\NodeElement')
+            ->disableOriginalConstructor()
+            ->getMock()
+        ;
+
+        $this->session
+            ->expects($this->exactly(1))
+            ->method('getPage')
+            ->will($this->returnValue($page))
+        ;
+
+        $page
+            ->expects($this->exactly(1))
+            ->method('find')
+            ->with('css', 'h2 > span')
+            ->will($this->returnValue($element))
+        ;
+
+        $element
+            ->expects($this->exactly(1))
+            ->method('hasAttribute')
+            ->with('blah')
+            ->will($this->returnValue(true))
+        ;
+
+        $this->assertWrongAssertion(
+          'elementAttributeNotExists',
+          ['css', 'h2 > span', 'blah'],
+          'Behat\\Mink\\Exception\\ElementHtmlException',
+          'The attribute "blah" was found in the element matching css "h2 > span".'
+        );
+    }
+
```

Comment:
It's better to rewrite tests for an `elementAttributeNotExists` method based on tests for an `elementAttributeExists` rather than copy/pasting them from an #709.

Reply:
Hey! This is a good idea, I updated the code with a test based on the `elementAttributeExists`, I'm sorry for copy and pasting the code from #709, my idea with that was to fix the code review from the PR.

Thanks!